client/src/pages/people/Show.js

```diff
@@ -174,6 +183,15 @@ const PersonShow = ({ pageDispatchers }) => {
           <a href={`mailto:${person.emailAddress}`}>{person.emailAddress}</a>
         )
 
+        const orderedFields = orderPersonFields()
+        const numberOfFieldsUnderAvatar =
+          Settings.fields.person.numberOfFieldsUnderAvatar || 3
+        const leftColumUnderAvatar = orderedFields.slice(
+          0,
+          numberOfFieldsUnderAvatar - 1
```

Comment:
`slice` excludes the end index, so that should be:
```
          numberOfFieldsUnderAvatar
```
You're missing a field now (and the number of fields under the avatar is 6 when the dictionary wants 7).